client/my-sites/earn/memberships/add-edit-plan-modal.jsx

```diff
@@ -260,14 +260,13 @@ const RecurringPaymentsPlanAddEditModal = ( {
 						{ translate( 'Select renewal schedule' ) }
 					</FormLabel>
 					<FormSelect id="renewal_schedule" value={ editedSchedule } onChange={ onSelectSchedule }>
-						<option value="1 month">{ translate( 'Monthly' ) }</option>
-						<option value="1 year">{ translate( 'Yearly' ) }</option>
+						<option value="one-time">{ translate( 'One time sale (non-recurring)' ) }</option>
```

Comment:
Can we drop `(non-recurring)`? I find "one time sale" quite clear, and an unusual term like "non-recurring" doesn't help here, I don't think.

Reply:
Thanks, fixed!